source/PlayerInfo.cpp

```diff
@@ -893,6 +854,57 @@ Account &PlayerInfo::Accounts()
 
 
 
+// Handle the daily salaries and payments.
+void PlayerInfo::DoAccounting()
```

Comment:
This should be accounting for the number of days that have passed, or else the message is going to be inaccurate (because it assumes only one day has passed).

Reply:
That would require too much refactoring imo. It seems a lot easier to add those extra messages to the message log (but not the list because that looks bad).

Reply:
Why would it require too much refactoring?

Reply:
Because I'd need to keep track of each transaction type (salaries, tributes, ships/outfits, mortgages, etc.) outside of DoAccounting()/Account::Step() to be able to compose a summary. (I cannot do all of these at once, because of event and mission actions triggering in the meantime.)

Reply:
Idk, it feels like it shouldn't be too hard to add up the various costs inside AdvanceDate (and removing DoAccounting again)?

Reply:
So should I only display a single message (when the entire AdvanceDate is done)?

Reply:
It would seem confusing otherwise I guess.

Reply:
I started doing this, but apparently Account::Step would also need to return the raw values instead of the formatted string. Is that alright?

Reply:
I think it is fine like this for now. The code is not wrong, it might just print a lot of messages if the player time travels for a lot of days.